test/run.sh

```diff
@@ -443,34 +438,6 @@ do
         diff_files="$diff_files $base.linked.wat"
     fi
   ;;
-  "did")
-    # The file is a .did file, so we are expected to test the idl
-    # Typecheck
-    $ECHO -n " [tc]"
-    didc --check $base.did > $out/$base.tc 2>&1
-    tc_succeeded=$?
-    normalize $out/$base.tc
-    diff_files="$diff_files $base.tc"
-
-    if [ "$tc_succeeded" -eq 0 ];
-    then
-      $ECHO -n " [pp]"
-      didc --pp $base.did > $out/$base.pp.did
-      sed -i 's/import "/import "..\//g' $out/$base.pp.did
-      didc --check $out/$base.pp.did > $out/$base.pp.tc 2>&1
```

Comment:
hmm, this test is still useful. Motoko needs the candid parser and it's good if we can test it somehow.

Reply:
Yes, this is the only thing we lose. We should import the new rust based didc and use it here, wdyt? Might be convenient to have that tool in the nix shell anyways.

Reply:
We want to test the parser from `idllib` which is used by import. Maybe we can wrap the .did file in the test as a  motoko program and test from moc --check

Reply:
Ah, this isn't the line where we check the output of `moc -idl`. Sorry.

We have existing tests that exercise the parser via `import` which is what matters. Should we go through the `.did` tests and see which are valuable and make them import tests?

Reply:
Sure, we can put all the `.did` file into the `import` test. I think they are all useful in testing the parser.